### PR TITLE
Update the game over message

### PR DESCRIPTION
**Pull request by hakohl**
Update the game over message so people know how to play again!

### CODE REVIEW

**Review by hakohl2 — APPROVED** [review]
I checked it. It is ok for me.

File: index.html [hunk 1]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```
Comment (hakohl2) [inline index.html:50]: no emogies, please

Comment (hakohl) [reply index.html:50]: I removed emogies.

**Review by hakohl — COMMENTED** [review]
ok

**Pull request merged**